CONTRIBUTING.md

```diff
@@ -41,6 +41,38 @@ make run-mock-target
 
 Now you can point your browser to `http://localhost:3000`
 
+### Using `docker-compose`
+
+We also provide a way to develop using `docker` container and orchestrated via composition.
+
+In the `Makefile` there's a section for `DOCKER COMPOSE`:
+
+- `make down`: Will call `docker-compose down` which will stop all running containers orchestrated with our `docker-compose` configuration
+- `make build-up`: Forces the building of the different images and then boots the 3 containers
+    + back-end
+    + front-end
+    + mockup-target
+- `make up`: Boots up the 3 containers without a force build. Changes to the front-end fails will livereload in the browser
```

Comment:
did you test the `livereload`? (I have had some issues with it in the past when running things inside docker containers)

Reply:
It was tested for sure.
I'll upload a video/gif later to the PR to show it working as I've had issues in the past as well with hot reload and containers